src/main/webapp/assets/styles/styles.css

```diff
@@ -71,13 +71,13 @@ footer {
 }
 
 .hero-image {
-	background: url('../img/heroEncuestas.jpg') no-repeat center center;
-	background-size: cover;
-	height: 40em;
-	color: white;
-	display: flex;
-	align-items: center;
-	justify-content: center;
+    background: url('../img/heroEncuestas.jpg') no-repeat center center !important;
```

Comment:
el  !important no se que es. ¿Se necesita?¿Se ha tocado este fichero? Si no se necesita no es necesario que subáis estos cambios. Simplemente no stageais el fichero cuando haceis el commit o incluso se puede hacer clic-derecho-> discard changes para que vuelva al estado original.
![0159c8c12c455c42872641f8c7c82879](https://github.com/user-attachments/assets/9c200438-ddb6-43dc-b1d8-55122088b379)